packages/scandipwa/src/component/Breadcrumb/Breadcrumb.component.js

```diff
@@ -78,9 +78,10 @@ export class Breadcrumb extends PureComponent {
 
     renderName() {
         const { name } = this.props;
+        const cleanName = name.replace('+', ' ');
```

Comment:
what if there will be more than 1 '+' symbol?

I think you need to use RegExp for this.